typescript/api-client/projects/vcd/schematics/src/schematics/plugin-seed/templates/10.0/__name@dasherize__/src/main/subnav-plugin.module.ts

```diff
@@ -0,0 +1,48 @@
+import { CommonModule } from "@angular/common";
+import { Inject, NgModule } from "@angular/core";
+import { RouterModule, Routes } from "@angular/router";
+import { Store } from "@ngrx/store";
+import { VcdApiClient, VcdSdkModule } from "@vcd/sdk";
+import { ExtensionNavRegistration, EXTENSION_ROUTE } from "@vcd/sdk/common";
+import { PluginModule } from "@vcd/sdk/core";
+import { TranslateService } from "@vcd/sdk/i18n";
+import { ClarityModule } from "clarity-angular";
```

Comment:
I don't believe this is still the package name for clarity components in the 0.13 case (which is the compatible version for 9.7 and 10.0). How does this project compile?